lighthouse-cli/index.js

```diff
@@ -127,7 +126,7 @@ if (cli.listTraceCategories) {
 
 const url = cli._[0];
 const outputMode = cli.output;
-const outputPath = cli.outputPath;
+const outputPath = cli['output-path'];
```

Comment:
might be a bug in yargs. others are translated into camelCase but output-path is not.

![image](https://cloud.githubusercontent.com/assets/39191/17380015/c39aaeba-5979-11e6-9618-46d9d1d82e28.png)